services/build-status.js

```diff
@@ -31,6 +32,7 @@ const otherStatuses = [
   'skipped',
   'starting',
   'stopped',
+  'testing',
```

Comment:
These three additional status (`infrastructure_failure` for a red status, and `initiated` and `testing` for other status) seemed like reasonable additions to the build status lists that wouldn't conflict with any other service (not that I could find, nor that I could envision). 

Let me know if anyone disagrees and/or if there's a reason we think these shouldn't be added